client/components/Video.jsx

```diff
@@ -11,14 +11,51 @@ class Video extends React.Component {
   componentDidMount() {
     // Begin animating the video when it starts playing
     const video = document.querySelector('video');
+    this.initializeVoice(video);
+
     video.addEventListener('canplay', (e) => {
       video.className += ' video-reveal';
       setTimeout(() => { video.className = 'video'; }, 2000);
     });
   }
 
-  emitPlayAndListenForPause(e) {
-    const video = e.target;
+  initializeVoice(video) {
+    // Initialize annyong voice command library and define commands
+    if (annyang) {
+      var commands = {
+        'play': () => {
+          console.log('play command received');
+          video.play();
+          this.emitPlayAndListenForPause(video);
+        },
+        'pause': () => {
+          console.log('pause command received');
+          video.pause();
+          this.emitPlayAndListenForPause(video);
+        },
+        'go back': () => {
+          console.log('go back command received');
+          // video.pause(); TODO: change this to a command to go back 5 seconds
+        }
+      };
+
+      annyang.debug();
```

Comment:
its helpful for now because we have not really proven yet that this works in production (production brings new issues due to https requirement). once we prove that it works, we can take out the debug statement